### DISCUSSION TITLE
Increase default hill shading magnitude value

### DISCUSSION THREAD

**Discussion by Sublimis** [mapsforge/mapsforge#1539] [Q&A] [answered]
How do you feel about increasing the default hill shading magnitude value for render themes, so that hills are more prominent by default? Currently it is hardcoded as 64 (out of 255), thus 25%. Some suggestions include: 96 (37.5%), 128 (50%), 160 (62.5%), 192 (75%). Maybe 128 would be a good start.

We're talking about line 280 of the `RenderThemeHandler.startElement()`:
```java
                short magnitude = 64;
```

There is the `magnitude` xml attribute of the `<hillshading>` render theme tag available, but of all the different render themes we've examined (motorider, tiramisu, elements, outdoor, activert, voluntary, velocity, black, ...) none use this attribute, they all rely on the defaults.

**Comment by devemux86** [comment, marked as answer]
@Sublimis

Thanks, this is indeed a good idea. :+1: 

I converted this to an issue:
- https://github.com/mapsforge/mapsforge/issues/1540
